### PR TITLE
TENDOFFICE-3447: Render data and implement all API properties

### PR DESCRIPTION
**Pull request by tu-ta**

### CODE REVIEW

File: wut-table/demo/index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      <h3>Basic wut-table demo</h3>
+      <demo-snippet>
+        <template>
+          <wut-table
+              item-link= "/page/id={{id}}&name={{name}}.html"
+              columns='["update", "id" , "name", "des"]'
+              columns-labels='{"update": "Last Update", "id": "ID", "name": "User Name", "des": "Description"}'
+              mobile-columns='["id", "name"]'
+              order='["update", "id", "des", "name"]'
+              data='[
```
Comment (farmer-katherine) [inline wut-table/demo/index.html]: @tu-ta can you update this example to use the static version of wut-model in the develop branch?

Comment (tu-ta) [inline wut-table/demo/index.html]: I will try to do it @farmer-katherine. But currently I see wut-model can't put data inside another element. May be I need your help.

**Comment by tu-ta** [comment]
@farmer-katherine I used wut-model on demo page

File: wut-table/demo/index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          </wut-model>
+          <wut-table
+                  rows-per-page= "{{data.rowsPerPage}}"
+                  item-link= "{{data.itemLink}}"
+                  columns="{{data.columns}}"
+                  columns-labels="{{data.columnsLabels}}"
+                  mobile-columns="{{data.mobileColumns}}"
+                  order="{{data.order}}"
+                  data="{{data.dataset}}"
+                  schema="{{data.schema}}"
```
Comment (farmer-katherine) [inline wut-table/demo/index.html]: Also @rpalmite this generally looks like we'd be adding boilerplate. It doesn't seem like we'd want to directly couple wut-table to wut-model - so would we put this mapping between the two in a tend-table?

So tend-table would look something like this:
`<tend-table model="user" model-schema="">`

We should probably discuss specifically what goes where. For example, it seems like item-link should probably be customizable by the view. So would we also pass that data through tend-table into wut-table?

Comment (farmer-katherine) [inline wut-table/demo/index.html]: I'm envisioning that "data" from the model will only include what is "data.dataset" in this code example.

rows-per-page, item-link, columns, column-labels, mobile-columns, order all seem like they are part of the view to me. 

The schema is not really part of the model or the view. I'm not clear where we would get the schema data itself - would it be file hosted in S3? Would it be an asset we published with bower?

File: wut-table/polymer.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "lint": {
+    "rules": [
+      "polymer-2-hybrid"
```
Comment (farmer-katherine) [inline wut-table/polymer.json]: @tu-ta Back this out. Stick with Polymer 1.0 for now.

File: wut-table/demo/index.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        <template is="dom-bind">
+          <wut-model data="{{data}}">
+            {
+              "rowsPerPage": "7",
+              "itemLink": "/page/id={id}&name={name}.html",
+              "columns": ["update", "id" , "name", "des"],
+              "columnsLabels": {"update": "Last Update", "id": "ID", "name": "User Name", "des": "Description"},
+              "mobileColumns": ["id", "name"],
+              "order": ["update", "id", "des", "name"],
+              "schema": {
```
Comment (rpalmite) [inline wut-table/demo/index.html]: should not be in data. use a html parameter!

File: wut-table/demo/index.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    </style>
+  </head>
+  <body>
+    <div class="vertical-section-container centered">
+      <h3>Basic wut-table demo</h3>
+      <demo-snippet>
+        <template is="dom-bind">
+          <wut-model data="{{data}}">
+            {
+              "rowsPerPage": "7",
```
Comment (rpalmite) [inline wut-table/demo/index.html]: should not be in data. use a html parameter!

File: wut-table/demo/index.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+              "itemLink": "/page/id={id}&name={name}.html",
```
Comment (rpalmite) [inline wut-table/demo/index.html]: should not be in data. use a html parameter!

File: wut-table/demo/index.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+              "columnsLabels": {"update": "Last Update", "id": "ID", "name": "User Name", "des": "Description"},
+              "mobileColumns": ["id", "name"],
+              "order": ["update", "id", "des", "name"],
+              "schema": {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "title": "User set",
+                "type": "array",
+                "items": {
+                "title": "User",
+                "type": "object",
```
Comment (rpalmite) [inline wut-table/demo/index.html]: should not be in data. use a html parameter!

File: wut-table/demo/index.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+  <body>
+    <div class="vertical-section-container centered">
+      <h3>Basic wut-table demo</h3>
+      <demo-snippet>
+        <template is="dom-bind">
+          <wut-model data="{{data}}">
+            {
+              "rowsPerPage": "7",
+              "itemLink": "/page/id={id}&name={name}.html",
+              "columns": ["update", "id" , "name", "des"],
```
Comment (rpalmite) [inline wut-table/demo/index.html]: should not be in data. use a html parameter!

File: wut-table/demo/index.html [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+              "columnsLabels": {"update": "Last Update", "id": "ID", "name": "User Name", "des": "Description"},
```
Comment (rpalmite) [inline wut-table/demo/index.html]: should not be in data. use a html parameter!

File: wut-table/demo/index.html (hunk 4 above)
Comment (rpalmite) [inline wut-table/demo/index.html]: should not be in data. use a html parameter!

**Comment by tu-ta** [comment]
@rpalmite, @farmer-katherine  I has just changed some data from wut-model to html parameters. And make datetime picker work. Take a look.

**Pull request merged**